tardis/io/model/parse_packet_source_configuration.py

```diff
@@ -46,7 +46,7 @@ def initialize_packet_source(packet_source, config, geometry):
     return packet_source
 
 
-def parse_packet_source_from_config(config, geometry, legacy_mode_enabled):
+def parse_packet_source_from_config(config, geometry, enable_legacy_mode):
```

Comment:
Why make this change?  Don't we usually want booleans to be in a format that reads like `flag is enabled` or `if flag_enabled`?